packages/cli/src/lib/builder/packager.ts

```diff
@@ -22,21 +22,23 @@ import { paths } from '../paths';
 import { makeConfigs } from './config';
 import { BuildOptions } from './types';
 
-function formatErrorMessage(error: any) {
+export function formatErrorMessage(error: any) {
   let msg = '';
 
   if (error.code === 'PLUGIN_ERROR') {
     if (error.plugin === 'esbuild') {
       msg += `${error.message}\n\n`;
-      for (const { text, location } of error.errors) {
-        const { line, column } = location;
-        const path = relativePath(paths.targetDir, error.id);
-        const loc = chalk.cyan(`${path}:${line}:${column}`);
+      if (error.errors) {
```

Comment:
Could we make this `if (error.errors?.length)` and also only add the double newlines if that condition is true? Just so we don't get the unnecessary whitespace.

Reply:
@freben Thanks! I've updated the code according to your suggestions.